crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -1053,36 +1068,39 @@ type SystemParameters {
 
 type TransactionBlock {
 	"""
-	The transaction block effects provide information such as gas cost and object changes
+	The effects field captures the results to the chain of executing this transaction
 	"""
 	effects: TransactionBlockEffects
 	"""
-	Sender of the transaction block
+	The address of the user sending this transaction block
 	"""
 	sender: Address
 	"""
-	The raw transaction block in BCS format
+	The SenderSignedData in BCS format
+	This is a struct that contains the transaction data
+	And a list of signatures of the signers of the transaction
 	"""
 	bcs: Base64
 	"""
-	The gas input of the transaction block
+	The gas input field provides information on what objects were used as gas
+	As well as the owner of the gas object(s) and information on the gas price and budget
+	If the owner of the gas object(s) is not the same as the sender, the tx block is a sponsored tx
 	"""
 	gasInput: GasInput
 	kind: TransactionBlockKind
 	"""
-	A list of signatures signed by all transaction participants.
-	1. Non-participant signatures must not be present.
-	2. Signature order does not matter
+	A list of signatures of all signers, senders, and potentially the gas owner if this is a sponsored tx
 	"""
 	signatures: [TransactionSignature]
 	"""
-	The digest of a transaction block is represented as a 32-byte array
-	This is returned to callers as a Base58 encoded string
+	The digest of a transaction block is a hashed 32-byte array of the TransactionData struct
```

Comment:
This makes it seem like the thing being hashed is a 32-byte array, rather than the other way around, how about:

```
A 32-byte hash that uniquely identifies the transaction block contents, encoded in Base58.
```